accord-core/src/main/java/accord/local/Node.java

```diff
@@ -128,6 +128,7 @@ public boolean equals(Object that)
 
         public boolean equals(Id that)
         {
+            if (that == null) return false;
```

Comment:
I forgot what hit this but likely was a test and not real code.  Equals is allowed to see `null` and is common to have `foo.equals(null)`